app/scripts/move-popup/dimMoveItemProperties.directive.js

```diff
@@ -186,5 +154,69 @@
         });
       }
     }
+
+    /*
+     * Determins if the item has damange and it's non kinetic.
+     *
+     * @param {Object} item - A item from the destiny server
+     * @return {Boolean}
+     */
+    function itemHasNonKineticDamage(item) {
+      if (!item.dmg || item.dmg === 'kinetic') {
+        return false;
+      }
+      return true;
+    }
+
+    /*
+     * Check for a lockable or elemental property in an item.
+     *
+     * @param {Object} item - A item from the destiny server
+     * @return {Boolean}
```

Comment:
Likewise on all these `@param` and `@return` statements - they take up space but don't educate. "has" or "is" in the method name is enough to know they return booleans.